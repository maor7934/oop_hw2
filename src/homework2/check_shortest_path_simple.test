CreateNode n1 1
CreateNode n2 1
CreateGraph A
AddNode A n1
AddNode A n2
AddEdge A n1 n2
FindPath A n1 -> n2
FindPath A n2 -> n1
