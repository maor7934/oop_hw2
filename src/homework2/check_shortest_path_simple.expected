created node n1 with cost 1
created node n2 with cost 1
created graph A
added node n1 to A
added node n2 to A
added edge from n1 to n2 in A
shortest path in A: n1 n2
no path found in A
